tools/mpremote/mpremote/main.py

```diff
@@ -337,7 +337,9 @@ def load_user_config():
     if path is None:
         path = os.getenv("HOME")
         if path is None:
-            return config
+            path = os.getenv("USERPROFILE")
```

Comment:
By now it would in my opinion be nicer to rewrite this like
```
for var in ["XDG_CONFIG_HOME", "HOME", "USERPROFILE"]:
  path = os.getenv(var)
  if path is not None:
    break
```

Reply:
makes sense 👍